include/boost/url/detail/any_allocator_base.hpp

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+
+#ifndef BOOST_URL_DETAIL_ANY_ALLOCATOR_BASE_HPP
+#define BOOST_URL_DETAIL_ANY_ALLOCATOR_BASE_HPP
+
+#include <boost/url/detail/type_id.hpp>
+
+#include <cstddef>
+
+namespace boost {
+namespace urls {
+namespace detail {
+
+struct any_allocator_base {
```

Comment:
can we please put the braces each on their own line (except for namespace declarations) why does this keep happening?